components/Landing/SmallFeatures/index.js

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+
+import SmallFeature from './SmallFeature';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    backgroundColor: theme.palette.primary.light,
+    borderRadius: theme.spacing(2),
+    overflow: 'hidden',
+    marginBottom: theme.spacing(15),
+  },
+}));
+
+const SmallFeatures = ({ features }) => {
+  const classes = useStyles();
+
+  return (
+    <>
+      <Typography variant="h3" align="center" gutterBottom>
+        And that&#39;s not all. We have more.
```

Comment:
what the rationale behind `&#39;`? Im pretty sure all devices nowadays know what an `'` apostrophe is. itll probably be easier to read in code too.

Reply:
eslint doesn't like that. It forces &#39; lol